eng/native/configurecompiler.cmake

```diff
@@ -176,6 +137,96 @@ elseif (CLR_CMAKE_HOST_UNIX)
   endif()
 endif(MSVC)
 
+if (CLR_CMAKE_ENABLE_SANITIZERS)
+  set (CLR_CMAKE_BUILD_SANITIZERS "")
+  set (CLR_CMAKE_SANITIZER_RUNTIMES "")
+  string(FIND "${CLR_CMAKE_ENABLE_SANITIZERS}" "address" __ASAN_POS)
+  if(${__ASAN_POS} GREATER -1)
+    # Set up build flags for AddressSanitizer
+    set (CLR_CMAKE_ENABLE_ASAN ON)
+    if (MSVC)
+      # /RTC1 is added by default by CMake and incompatible with ASAN, so remove it.
+      string(REPLACE "/RTC1" "" CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
+      string(REPLACE "/RTC1" "" CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG}")
+      string(REPLACE "/RTC1" "" CMAKE_SHARED_LINKER_FLAGS_DEBUG "${CMAKE_SHARED_LINKER_FLAGS_DEBUG}")
+      string(REPLACE "/RTC1" "" CMAKE_EXE_LINKER_FLAGS_DEBUG "${CMAKE_EXE_LINKER_FLAGS_DEBUG}")
+    endif()
```

Comment:
This was done above on lines 119-123. Is this needed in both places?

Reply:
I meant to remove that one, good catch!